src/Core/Data/RecipeReview.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class RecipeReview
+	{
+
+		public UserProfile Author { get; set; }
+
+		public string Comments { get; set; }
+		
+		public bool AuthorBrewedIt { get; set; }
+			
+
```

Comment:
I think it should have an ID and parent ID pointing back to the recipe it relates to.